clusters/app.ci/prow/03_deployment/qe_private_deck.yaml

```diff
@@ -125,6 +125,7 @@ objects:
                 - --kubeconfig-dir=/etc/build-farm-credentials
                 - --kubeconfig-suffix=config
                 - --tenant-id=qe-private
+                - --tenant-id=gangway-api
```

Comment:
That's nice but I guess we need to add it as well to our deck. This will result in seeing qe jobs triggered by gangway in public deck as well, right? And vice versa? The solution for that could be adding 2 gangway tenants (if possible). One for public jobs and second one for QE.

Reply:
The `tenant_id` isn't actually used for that much in `deck`. It merely provides filtering on the `prowjobs` that are visible. The actual `gangway-api` tenanted jobs won't show in public deck as their logs are still stored in the private bucket that public deck has no access to. Even now, the `gangway-api` tenanted jobs from the qe-private repos are not appearing in public deck.

As for public deck needing the `gangway-api` id added, I can do some additional research and see if that is necessary.

Reply:
It does appear that `gangway-api` tenanted jobs are not showing at all in public deck. I will have to add the tenant id, **and** the default tenant id there as well as deck-internal.